extensions/amp-story/0.1/amp-story.css

```diff
@@ -268,6 +268,22 @@ amp-story:not([desktop]) >
   z-index: 2 !important;
 }
 
+/** Call to action layer */
+amp-story-cta-layer {
+  bottom: 0 !important;
+  display: block !important;
+  height: 20% !important;
+  left: 0 !important;
+  position: absolute !important;
+  right: 0 !important;
+  z-index: 3 !important;
+}
+
+amp-story-cta-layer * {
```

Comment:
I think we can probably drop this whole style; it's applied to grid layer for legacy reasons.

Reply:
Done.